### PR TITLE
SelectBox: input should not be cleared after click on the toolbar (T897239)

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: js/ui/select_box.js [hunk 1]
```diff
@@ -232,6 +233,7 @@ const SelectBox = DropDownList.inherit({
     _createPopup: function() {
         this.callBase();
         this._popup.$element().addClass(SELECTBOX_POPUP_CLASS);
+        this.$element().find(`.${OVERLAY_CONTENT_CLASS}`).attr('tabindex', -1);
```
Comment (DokaRus) [inline js/ui/select_box.js]:
```suggestion
```

Let's try to extend Popup config:

```js
_popupConfig: function() {
        return extend(this.callBase(), {
            tabIndex: -1
        });
    },
```

Comment (ksercs) [reply js/ui/select_box.js]: It does not work, because popupConfig is extended width dropDownOptions cache on the dropDownEditor level

Comment (DokaRus) [reply js/ui/select_box.js]: So 
```suggestion
        this._popup.option('tabIndex', -1);
```

Comment (ksercs) [reply js/ui/select_box.js]: fixed

File: js/ui/select_box.js [hunk 2]
```diff
@@ -232,6 +232,7 @@ const SelectBox = DropDownList.inherit({
     _createPopup: function() {
         this.callBase();
         this._popup.$element().addClass(SELECTBOX_POPUP_CLASS);
+        this._popup.$content().parent().attr('tabindex', -1);
```
Comment (DokaRus) [inline js/ui/select_box.js]:
```suggestion
        this._popup.overlayContent().attr('tabindex', -1);
```

Comment (ksercs) [reply js/ui/select_box.js]: fixed

**Review by DokaRus — APPROVED** [review]

**Review by San4es — APPROVED** [review]

**Pull request merged**